crates/wysiwyg/src/composer_model.rs

```diff
@@ -354,6 +436,61 @@ where
             panic!("Trying to bold a non-text node")
         }
     }
+
+    fn add_list_item(&mut self, handle: DomHandle, location: usize) {
+        let list_node = self.state.dom.lookup_node_mut(handle);
+        if let DomNode::Container(list) = list_node {
+            list.append(DomNode::new_list_item(
+                "li".to_html(),
+                vec![DomNode::Text(TextNode::from("\u{200B}".to_html()))],
+            ));
+            self.state.start = Location::from(location + 1);
```

Comment:
I think it would be good to select the hidden space, so should `self.state.start` be set to `location` instead of `location + 1`?

Reply:
I got a slightly weird selection issue with this on iOS, while this could probably be fixed with a platform-specific workaround, there is also the issue where we move the cursor somewhere else then move back to this TextNode...
Rather than reselecting the hidden space, it feels like it could be better to have some mechanic to automatically remove the hidden space when inserting near it ? (or even keeping it should be fine)

Reply:
Yeah. I was also wondering about just deleting all zero-width spaces whenever we convert to HTML.

Fine to leave it for now. When we add testcases all this is going to work itself out.